server/coordinator/node_picker_test.go

```diff
@@ -29,7 +30,7 @@ func TestRandomNodePicker(t *testing.T) {
 
 	for i := 0; i < nodeLength; i++ {
 		nodes = append(nodes, metadata.RegisteredNode{
-			Node:       storage.Node{Name: strconv.Itoa(i), State: storage.NodeStateOffline},
+			Node:       storage.Node{Name: strconv.Itoa(i), LastTouchTime: uint64(time.Now().Unix() - int64(time.Minute))},
```

Comment:
Why not do `uint64(time.Now().Unix() - int64(time.Minute))` just once and use it everywhere?